### ISSUE TITLE
Is it possible to only show the value of the currently held item?

### ISSUE THREAD

**Issue by Ghostypunchy** [DeathGOD7/LCItemValue#1]
I don't know if it's possible or not, but how possible would it be to add the ability to only show the value of the currently held item? 

either that or the ability to display all item values separately instead of only the total sum. Or even both with the option to choose.
this would make comparing scrap a lot easier without having to constantly drop items to compare prices. 

Happy New Year!

**Comment by RexSE7EN** [comment]
I think it is possible to make it show price separately or only on the held ones.

A bar showing price below the item individually or on the right side as list would be ok I guess.

The thing is...when to show it, to prevent gui clutter is problem. Either it only shows when scanning or visible always or until the item is dropped.

Happy New Year!!

**Comment by Ghostypunchy** [comment]
is it possible to implement both with a config file? have it be on scan by default, with the option to have it always on. 

i think thats a good place to start, and then go from there!

**Comment by RexSE7EN** [comment]
Yes it is possible to implement both but where to show is the issue.

Possible is above hotbar, inside hotbar (either top or bottom) or at the top right like the price tag (like how the "NEW" label is shown in products).

**Comment by Ghostypunchy** [comment]
i think for better clarity, if possible, is to show each items value above each item in the hotbar.

**Comment by ByblackSun** [comment]
> 是的，兩者都可以實現，但在哪裡顯示是問題。
> 
> 可能是在熱欄上方、熱欄內部（頂部或底部）或右上角，如價格標籤（如「新」標籤在產品中的顯示方式）。

I want it to only show up when there is an item on hand that needs to be scanned.
